Cesium.Runtime/StdLibFunctions.cs

```diff
@@ -33,14 +34,68 @@ public static void SRand(uint seed)
 
     public static int System(byte* command)
     {
-        switch (StdIoFunctions.Unmarshal(command))
+        string? shellCommand = StdIoFunctions.Unmarshal(command);
+        var startParameters = RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+            ? new ProcessStartInfo("cmd", $"/c {shellCommand}")
+            : new ProcessStartInfo("/bin/sh", $"-c \"{shellCommand}\"");
+        Process? pa11yCiInstallation = Process.Start(startParameters);
+        if (pa11yCiInstallation is null)
```

Comment:
On Windows at least, you can set a `UseShellExecute` flag instead.

And for the Unix version, let's pass the arguments via `ArgumentList` instead of adding quotes: that's more reliable.

Reply:
ArgumentList is not in netstandard2.0, do we want code split here?

Reply:
and how do you want to use `UseShellExecute` ? Are you want split command line in parts to get executable name ?

Reply:
Ok, after reading the docs, I no longer think `UseShellExecute` is of any help for us here.

Also, now I see that we can't use `ProcessStartInfo.ArgumentList` on .NET Standard 2.0. Sigh. Okay, then.